contracts/stellar-interchain-token-service/src/tests/executable.rs

```diff
@@ -116,7 +116,7 @@ fn interchain_transfer_execute_succeeds() {
 
     let amount = 1000;
     let deployer = Address::generate(&env);
-    let (token_id, _) = setup_its_token(&env, &client, &deployer, amount);
+    let (token_id, _) = setup_its_token(&env, &client, &deployer, amount, &None);
```

Comment:
Passing `&None` is ugly. Avoid using references to `Option` and pass by value instead